devcareer/code-jammers-backend
# #174691368 Fix test coverage and add maintainability badges

### What does this PR do?

This PR fixes the test coverage badge and adds a maintainability badge.

### Description of tasks to be completed?

-  Fix test coverage bade
-  Add Maintainability badge to the readMe file
   
### How should this be manually tested/checked?
- Check the `checks` section at the bottom of the pull request to see the result of codeclimate checks.
- Check the `readMe` of this branch `fx-test-coverage-maintainability-174691368` to see the badges.
- Check the `Actions` tab to see the result of the build.

### What are the relevant pivotal tracker stories?
[#174691368 ](https://www.pivotaltracker.com/story/show/174691368)
